packages/design-system/src/components/Autocomplete/Autocomplete.test.tsx

```diff
@@ -19,8 +19,34 @@ function makeAutocomplete(customProps = {}) {
 describe('Autocomplete', () => {
   it('renders Autocomplete component', () => {
     const { container } = makeAutocomplete();
-    const page = container.firstChild;
-    expect(page).toMatchSnapshot();
+
+    const wrapperEl = container.querySelectorAll('.ds-c-autocomplete');
+    expect(wrapperEl.length).toEqual(1);
+
+    const input = screen.getByRole('combobox');
+    expect(input).toBeInTheDocument();
+    expect(input).toHaveAttribute('aria-autocomplete', 'list');
+    expect(input).toHaveAttribute('aria-controls');
+    expect(input).toHaveAttribute('aria-activedescendant');
+    expect(input).toHaveAttribute('aria-expanded', 'false');
+    expect(input).toHaveAttribute('role', 'combobox');
+    expect(input).toHaveAttribute('type', 'text');
+
+    const menuEl = container.querySelector('.ds-c-autocomplete__menu-container');
+    expect(menuEl).toBeInTheDocument();
+    expect(menuEl).toHaveAttribute('hidden');
+
+    const list = screen.queryByRole('listbox');
+    expect(list).not.toBeInTheDocument();
```

Comment:
These are really thorough! One step further for `aria-controls` would be `toHaveAttribute('aria-controls', list.id)`